src/functionsDeployHelper.ts

```diff
@@ -211,8 +211,8 @@ export function printSuccess(funcName: string, type: string) {
 }
 
 export async function printTriggerUrls(projectId: string, sourceUrl: string) {
-  const functions = await cloudfunctions.listAllFunctions(projectId);
-  const httpsFunctions = functions.filter((fn) => {
+  const res = await cloudfunctions.listAllFunctions(projectId);
```

Comment:
Separately, why is this needed? We already have the results from create and update.

Reply:
Good point, this is probably an unnecessary call.